lib/cog/adapters/hipchat/api.ex

```diff
@@ -38,6 +38,10 @@ defmodule Cog.Adapters.HipChat.API do
     end
   end
 
+  def lookup_user("@" <> handle),
+    do: lookup_user(handle: handle)
+  def lookup_user(handle) when is_binary(handle),
+    do: lookup_user(handle: handle)
```

Comment:
I'm not a huge fan of the different arg types here, but I can live with it. Ideally we'd either have another function like `lookup_user_by_formatted_handle` (with a shorter name) for this. I can deal with it though.